uaa/remove-user-guid-from-all-groups.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+usage() {
+  echo "Usage: "
+  echo "  $0 <user-guid>"
+  echo " "
+  echo "  <user-guid>: The guid of the user to remove from all groups."
```

Comment:
`The GUID of the user to remove from all groups (up to 100, pagination not yet built-in)`

Reply:
Not sure what you are commenting on.

Reply:
Actually this is more of an issue than I initially thought. 

I was commenting on the usage text since, it would only remove the user from the first 100 groups returned by the `uaac curl` call. I don't know if we're close to that limit or not. 

It may be worth adding pagination if we're over, say, 50 groups total and may hit 100 in the next few years.  

(I initially thought it was only an issue if the user themself was in more than 100 groups, but I see that I was wrong there)